ES/ESmy018.xml

```diff
@@ -260,19 +261,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <persName ref="PRS13125ZabanaKrestos">Zabāna Krǝstos</persName>.</note>
     </msItem>
 </msItem>
+                    </msItem>
 
  <msItem xml:id="ms_i2">
- <locus from="44ra" to="60va"/>
- <title type="complete" ref="LIT2085OntheL">Discourse by John Chrysostom on the life and
-decollation of John the Baptist</title>
-<incipit xml:lang="gez">
-<locus target="#44ra"/><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወ</hi>መንፈስ፡ ቅዱስ፡ አሐዱ፡ <hi rend="rubric">አምላክ፡
-ዝንቱ፡ ነገር፡ በከ</hi>መ፡ ኮነ፡ ሕይወቱ፡ ለቅዱስ፡ <hi rend="rubric">ዮሐንስ፡ መጥምቅ፡ ወከ</hi>መ፡ እፎ፡
-ኮነ፡ ምትረተ፡ ርእሱ፡ ወተርጐሞ፡ ቅዱስ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ በትርያርክ፡ ሊቀ፡ ጳጳሳት፡ ብሂል፡ ዘሀገረ፡ ቈስጠንጥንያ፤ ወሶበ፡ ኮነ፡ ስደቱ፡ ውስተ፡ አብራክይስ፡ አመ፡ ተስእሎ፡ አይሁዳዊ፡
-ኬልቄዶናዊ፡ ዘአምነ፡ በእግዚእነ፡ ኢየሱስ፡ ክርስቶስ።
-</incipit>
-
-                     <msItem xml:id="ms_i3">
+<title>Works dedicated to John the Baptist</title>
```

Comment:
They can be nested as a collection, or be separate items, but in any case after the msItem 1 was closed. The collection solution fits better as the colophon at the end belongs to the whole set of texts and the colophon must be within an msItem

Reply:
Ok, we can call it a collection, even if to do this we would need to find at least one more manuscript with the same set of texts.

Reply:
we don't have to, we are not creating any work ID, we simply nest the texts together  - you must have a wrapping msItem anyway to assign the colophon to the unit

Reply:
Please approve and merge after reviewing